dashboard/app/controllers/report_abuse_controller.rb

```diff
@@ -107,6 +114,15 @@ def report_abuse_form
       name: current_user&.name,
       email: current_user&.email,
       age: current_user&.age,
+      requireCaptcha: require_captcha?,
+      captchaSiteKey: CDO.recaptcha_site_key,
     }
   end
+
+  private
+
+  def require_captcha?
+    return false if current_user&.admin? || current_user&.project_validator?
+    true
+  end
```

Comment:
@davidsbailey is this another one that should be moved to cancan? I wasn't sure, because the use case here is a little different

Reply:
I agree that it looks different -- I wouldn't worry about moving that into cancan.